src/libcore/ptr.rs

```diff
@@ -946,6 +946,58 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
     intrinsics::volatile_store(dst, src);
 }
 
+/// Freezes `count * size_of::<T>()` bytes of memory, converting undefined data into
+/// arbitrary but fixed data.
```

Comment:
Could we maybe add a quick sentence that says that already-initialized data is not modified. I know that this is implied by the later text but it would be nice for it to be explicitly stated.

Reply:
Done